ifit/undisposed-fody
# Mob 886 update fody

Updating our `undisposed-fody` fork with the latest commits from [fody](https://github.com/ermshiperete/undisposed-fody).

- https://github.com/ifit/wolf/pull/6615
- https://github.com/ifit/shire/pull/3298

### Testing
1. Download locally
1. Run `build.sh`
1. Verify that NuGet package is created
